modules/generators/flag/flag_words/secgen_local/local.rb

```diff
@@ -2,17 +2,20 @@
 require_relative '../../../../../lib/objects/local_string_generator.rb'
 require_relative '../../../../../lib/helpers/blacklist.rb'
 class WordFlagGenerator < StringGenerator
+  attr_accessor :counter
+
   def initialize
     super
     self.module_name = 'Random Word Based Flag Generator'
+    self.counter = 1
```

Comment:
The default should remain 4, otherwise if this module is randomly selected a flag could be as simple as `flag{flag}`